src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+.bar {
+  font-family: Avenir, sans-serif;
+  font-weight: bold;
+  color: #3d4e61;
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.firstBar {
+  font-size: 16px;
+  padding: 25px 0 23px 62px;
+}
```

Comment:
checklist point 2
you can write these styles at .bar, remove styles for .firstBar and leave styles for .secondBar